config/initializers/filter_parameter_logging.rb

```diff
@@ -32,5 +34,5 @@
 # Use this to limit dissemination of sensitive information.
 # See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += %i[
-  passw secret token _key crypt salt certificate otp ssn
+  passw email secret token _key crypt salt certificate otp ssn cvv cvc
```

Comment:
I suggest we filter for `mail` here instead, as the attribute on the `User` model is called `mail`. This might also future-proof us for some variants, such as `e_mail`.

```suggestion
  passw mail secret token _key crypt salt certificate otp ssn cvv cvc
```